static/d3-aula.js

```diff
@@ -266,7 +266,7 @@
         // be slightly nicer to remember which nodes were invisible
         // and recover the state before the previous click on the root
         // node.)
-        var on_click = function(d) {
+        var on_click = function(clickee) {
```

Comment:
this is the node that is clicked on, not the event.  if anything, we could call it delegate.  i will change that.